src/replays/types.ts

```diff
@@ -7,6 +7,7 @@ export interface FileResult {
   startTime: string | null;
   lastFrame: number | null;
   metadata: MetadataType | null;
+  winners: number[];
```

Comment:
```suggestion
  winnerIndices: number[];
```
last nit: can we call these indices so it's a bit clearer as to what the number is. like it's not port but player index.